response_builder.go

```diff
@@ -63,6 +64,11 @@ func (rb *responseBuilder) SetFragments(results []*multiplexer.Result) {
 	}
 }
 
+func (rb *responseBuilder) SetDuration(duration int64) {
+	outputHtml := bytes.Replace(rb.body, []byte("<view-proxy-timing></view-proxy-timing>"), []byte(strconv.FormatInt(duration, 10)), 1)
```

Comment:
```suggestion
	outputHtml := bytes.Replace(rb.body, []byte("<view-proxy-timing></view-proxy-timing>"), []byte(strconv.Itoa(duration)), 1)
```

Minor, but I think `Itoa` is a shortcut for `FormatInt(int64, int)`. Thoughts?

Reply:
I couldn't get `Itoa` to work with `int64` But it does work with `int`.